cmd/sensuctl/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"os"
+
+	"github.com/sensu/sensu-go/cli"
+	"github.com/sensu/sensu-go/cli/commands"
+	hooks "github.com/sensu/sensu-go/cli/commands/hooks"
+	"github.com/sensu/sensu-go/cli/commands/root"
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	rootCmd := root.Command()
+	sensuCli := cli.New(rootCmd.PersistentFlags())
+
+	rootCmd.PersistentPreRunE = func(cmd *cobra.Command, _ []string) error {
+		return hooks.ConfigurationPresent(cmd, sensuCli)
+	}
+
+	commands.AddCommands(rootCmd, sensuCli)
+
+	if err := rootCmd.Execute(); err != nil {
+		os.Exit(1)
```

Comment:
log.Fatal(err) may be better here

Reply:
Cobra already returns the error, using logger.Fatal would provoke the following result:

```
$ sensuctl version2
Error: unknown command "version2" for "sensuctl"

Did you mean this?
	version

Run 'sensuctl --help' for usage.
FATA[0000] unknown command "version2" for "sensuctl"

Did you mean this?
	version
  component=sensuctl
```